### ISSUE TITLE
A few errors to fix

### ISSUE THREAD

**Issue by chamborz** [kolton/d2bot-with-kolbot#2425]
Hi guys i have a few error but i dont know how to fix it, i need help 
I have to bots running, lightsorc and hammerdin

Log on sorc:

Error in Diablo (diablo.js #326) Diablo not found (Area: 108
Error in SharpTooth (attack.js #229) Attack.kill: Target not found (Area: 111
Error in Mephisto (mephisto.js #108) Failed to lure Mephisto. (Area: 102
Crash Info: Script: Worldstone Area: Worldstone Keep Level 3
Error in SharpTooth (attack.js #298) Failed to kill Sharptooth Slayer (No valid attack skills) (Area: 111
Error in Pindleskin (attack.js #298) Failed to kill Nihlathak (No valid attack skills) (Area: 124
Crash Info: Script: no entry Area: Rogue Encampment

Log on Hammerdin

Error in Andariel (andariel.js #41) Failed to move to Catacombs Level 4 (Area: 35, 
Error in AncientTunnels (ancienttunnels.js #21) Failed to move to Ancient Tunnels (Area: 44
Error in Summoner (summoner.js #25) Failed to move to Summoner (Area: 74, 
Error in Duriel (duriel.js #55) Failed to move to Tal Rasha's Tomb (Area: 46
Error in Mephisto (mephisto.js #138) Failed to move to Durance Level 3 (Area: 101
Error in Diablo (diablo.js #435) Failed to move to Chaos Sanctuary (Area: 107, 
Error in Pindleskin (pindleskin.js #18) Failed to move to Nihlahak's Temple (Area: 123
Error in Eldritch (attack.js #435) Attack.clear: Shenk the Overseer not found (Area: 111
Error in Eyeback (eyeback.js #13) Failed to move to Eyeback the Unleashed (Area: 112, 
Error in SharpTooth (sharptooth.js #13) Failed to move to Sharptooth Slayer (Area: 111, 
Error in ThreshSocket (threshsocket.js #13) Failed to move to Thresh Socket (Area: 112, 
Error in Frozenstein (frozenstein.js #13) Failed to move to Frozenstein (Area: 113
Error in Bonesaw (bonesaw.js #13) Failed to move to Bonesaw (Area: 115, 
Error in Snapchip (snapchip.js #13) Failed to move to Snapchip Shatter (Area: 118,
Error in Baal (baal.js #208) Failed to move to Throne of Destruction. (Area: 129, 
Error in Diablo (diablo.js #200) Failed to open Vizier seals. (Area: 108

I think that the hammerdin error are on the same run, every 2 or 3 hours give me thats error, the sorc are more unstable and give me a error every run

what i can to do guys? any idea?

**Comment by DarkHorseDre** [comment]
Most of this is common and config/spec dependent (e.g. hammerdins need tele and be able to clear quickly (with good FHR & FCR) to avoid being blocked whilst moving and interacting with things):

Diablo not found - when the bot THINKS it has cleared all mobs and opened all seals it looks for Dia to appear - it fails sometimes - not sure of the current status of this on the repo but you can check by searching

attack.kill target not found - bot has arrived at target and it is not there (or clearpath is enabled and it killed the target before exiting pather.js and continuing the bot script) - nothing you can do except disable clearpath if enabled (or change it to only attack normal mobs)

lure Mephisto - this is enabled in the Mephisto section of char config - Mephisto luring/moat trick failed - not much you can do except to improve the code yourself

crashes - reason for many crashes have not been found/implemented yet

no valid attack skills - your attack config does not allow the bot to kill the target (I assume because of immunities and the lack of assigned skills for that immunity - telestomp should be enabled if this is the case (and the bot needs to be able to eventually kill the boss without dying)

no entry to rogue encampment - this is a bug and IIRC someone has fixed this in a post - search for it

failed to move - something blocks the bot (usually a mob) or it gets stuck on an object.  it's probably being surrounded by mobs and cant click the entrance (especially if it doesn't have tele)

failed to open seals - IIRC someone else opened the seal or the seal is bugged - doubt there's much you can do.

in short, check your config and check to see if these issues have been raised before and take the solutions where provided.
